include/psp2kern/kernel/sysmem.h

```diff
@@ -203,6 +203,10 @@ int ksceKernelGetPaddr(void *addr, uintptr_t *paddr);
 
 int ksceSysrootIsManufacturingMode(void);
 
+int ksceDebugPrintf(const char *fmt, ...);
+
+int ksceDebugPrintf2(int unk0, int unk1, const char *fmt, ...);
```

Comment:
Please add a line of doc where you give known working valies for the unk arguments. This saves frustration from developers willing to use this function.